### DISCUSSION TITLE
How to "compact" migration after development phase?

### DISCUSSION THREAD

**Discussion by sotomski** [serverpod/serverpod#2545] [Q&A] [answered]
Hi all, my project is closing on to the big release (yay! 🎉) and I'm preparing the prod environment. 
I've created tons of migrations during development stage which I don't really need anymore. I'm looking for a way to tidy up a little bit and merge (delete?) all of them so that I have the project in pristine condition for the release. 

Ani idea how to do that pain-free?

**Comment by Isakdl** [comment, marked as answer]
Hello @sotomski 

If you do not have any data in production yet you should be safe just to delete all migrations and create a clean new one. Then wipe your development database and apply this new migration.

This will only be painful if you want to keep data in your database.
